{"frames":[{"func":"cfa(m_cfa_well, data = df_well)","file":"","line_number":0,"end_line_number":0,"character_number":0,"end_character_number":0},{"func":"eval(mc, parent.frame())","file":"","line_number":0,"end_line_number":0,"character_number":0,"end_character_number":0},{"func":"eval(mc, parent.frame())","file":"","line_number":0,"end_line_number":0,"character_number":0,"end_character_number":0},{"func":"lavaan::lavaan(model = m_cfa_well, data = df_well, model.type = \"cfa\", \n    int.ov.free = TRUE, int.lv.free = FALSE, auto.fix.first = TRUE, \n    auto.fix.single = TRUE, auto.var = TRUE, auto.cov.lv.x = TRUE, \n    auto.cov.y = TRUE, auto.th = TRUE, auto.delta = TRUE, auto.efa = TRUE)","file":"","line_number":0,"end_line_number":0,"character_number":0,"end_character_number":0}],"message":"Error in lavaan::lavaan(model = m_cfa_well, data = df_well, model.type = \"cfa\",  : \n  object 'df_well' not found\n"}